core/org.osate.xtext.aadl2.ui/src/org/osate/xtext/aadl2/ui/outline/Aadl2OutlineTreeProvider.java

```diff
@@ -140,7 +146,9 @@ protected boolean isLeaf(final EObject modelElement) {
 				|| modelElement instanceof FlowSpecification || modelElement instanceof FlowImplementation
 				|| modelElement instanceof EndToEndFlowImpl || modelElement instanceof Property
 				|| modelElement instanceof PropertyConstant || modelElement instanceof PropertyType
-				|| modelElement instanceof Connection) {
+				|| modelElement instanceof DataType || modelElement instanceof AbstractSubcomponent
```

Comment:
Here you are saying that data types and abstract subcomponents should never be expanded. Why?

Reply:
i think this was supposed to not expand elements like gps1 in integration.aadl in speed regulation example. But it looks like it is unnecessary - outline view does not show them as expanded, so it is already handled by other code check